### PR TITLE
Fix a few legal things

### PR DESCRIPTION
**Pull request by richardfontana**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit 672973231849a084b14a6194c7da62eede929762:
:godmode: All seems good! :cake: :punch: :cookie: 
*CFME QE Bot*

File: data/bundles/merkyl/LICENSE [hunk 1]
```diff
@@ -0,0 +1,19 @@
+Copyright (c) 2012, Marcel Hellkamp.
```
Comment (psav) [inline data/bundles/merkyl/LICENSE]: Can we change the name of this file to LICENSE-bottle and update the header in the bottle file? As merkyl is actually written by us, and are just including bottle for convenience here?

Comment (richardfontana) [inline data/bundles/merkyl/LICENSE]: Yes, that's okay.

Comment (psav) [inline data/bundles/merkyl/LICENSE]: Awesome thanks

Comment (psav) [inline data/bundles/merkyl/LICENSE]: I'll open a new PR on this in a bit

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by psav** [comment]
Closed via #2253

**Pull request closed without merging**